.circleci/config.yml

```diff
@@ -134,7 +134,7 @@ jobs:
         name: Build Docker image
         command: |
           echo $MASTER_KEY > config/master.key
-          ./bin/build
+          bundler exec ./bin/build
```

Comment:
this is a bash script, not a ruby script. I think you might want to put that here: https://github.com/sul-dlss/rialto-webapp/blob/master/bin/build#L13 instead.